resources/rest-service/cloudify/migrations/versions/396303c07e35_5_2_to_5_3.py

```diff
@@ -18,19 +21,55 @@
 branch_labels = None
 depends_on = None
 
+installation_status = sa.Enum(
+    'active',
+    'inactive',
+    name='installation_status'
+)
+
+deployment_status = sa.Enum(
+    'good',
+    'in_progress',
+    'require_attention',
```

Comment:
(very minor) This should probably be 'requires_attention' or maybe 'attention_required'

Reply:
well, this is how the Ofer want to have it like this

Reply:
Hmm. It feels clumsily phrased but sure, let's get this merged and I'll raise it with him.